src/ImageSharp/Processing/Processors/Convolution/Convolution2PassProcessor{TPixel}.cs

```diff
@@ -42,12 +42,12 @@ internal class Convolution2PassProcessor<TPixel> : ImageProcessor<TPixel>
         }
 
         /// <summary>
-        /// Gets the horizontal gradient operator.
+        /// Gets the horizontal convolution kernel.
         /// </summary>
         public DenseMatrix<float> KernelX { get; }
 
         /// <summary>
-        /// Gets the vertical gradient operator.
+        /// Gets the vertical convolution kernel.
         /// </summary>
         public DenseMatrix<float> KernelY { get; }
```

Comment:
Small side note, I'm still convinced that using `DenseMatrix<T>` for separable 1D kernels makes the code less intuitive (those are 1D vectors, not matrices) and potentially more overhead for the unnecessary 2D coordinate calculation. I think we could possibly look into having a different DenseVector<T> type in the future that would just map to a 1D vector, to use in cases such as this.

Reply:
It makes my life easier as I can use the same code throughout when doing things like offset mapping. 

We also use the same code for the non-separable convolution operation so that can be arbitrary dimensions.